packages/back-end/src/app/controllers/easy-genomics/user/create-bulk-user-invitation-requests.lambda.ts

```
@@ -29,13 +30,14 @@ export const handler: Handler = async (
     // Data validation safety check
     if (!CreateBulkUserInvitationRequestSchema.safeParse(request).success) throw new Error('Invalid request');
 
-    // Check if Organization & User records exists
-    const organization: Organization = await organizationService.get(request.OrganizationId); // Throws error if not found
-
-    if (!organization) {
-      throw new OrganizationNotFoundError();
```

this check has been removed, is it not needed any more?

---

It's not needed, since the service layer is already throwing the exception rather than returning null.